x-pack/plugins/apm/public/components/shared/Links/apm/service_overview_link.tsx

```diff
@@ -9,19 +9,34 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import React from 'react';
+import { useUrlParams } from '../../../../context/url_params_context/use_url_params';
+import { pickKeys } from '../../../../../common/utils/pick_keys';
+import { APMQueryParams } from '../url_helpers';
 import { APMLink, APMLinkExtendProps, useAPMHref } from './APMLink';
 
 interface ServiceOverviewLinkProps extends APMLinkExtendProps {
   serviceName: string;
 }
 
+const persistedFilters: Array<keyof APMQueryParams> = [
+  'latencyAggregationType',
+];
+
 export function useServiceOverviewHref(serviceName: string) {
-  return useAPMHref(`/services/${serviceName}/overview`);
+  return useAPMHref(`/services/${serviceName}/overview`, persistedFilters);
```

Comment:
This will overwrite the default `persistedFilters` eg time range, environment etc. Is that intentional?

Reply:
@sqren there are a few default fields that are merged with the item I added to this `persistedFilters`, take a look at: https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/public/components/shared/Links/apm/APMLink.tsx#L66

Reply:
Ah, `PERSISTENT_APM_PARAMS` is applied twice. Can you remove it from `useAPMHref` to avoid any confusion?